cranelift-codegen/src/regalloc/coloring.rs

```diff
@@ -226,7 +226,22 @@ impl<'a> Context<'a> {
                 lv.affinity.display(&self.reginfo),
                 self.cur.func.locations[lv.value].display(&self.reginfo)
             );
-            if let Affinity::Reg(rci) = lv.affinity {
+            if let Affinity::RegUnit(reg) = lv.affinity {
+                let bank = self.reginfo.bank_containing_regunit(reg).unwrap();
```

Comment:
This is just the body of `spilling::toprc_containing_regunit` - it seems like that function could/should be moved over to `RegInfo`, but also not sure if I'm using it terribly wrong :)

Reply:
went ahead and did this because looking at this again it feels like an obvious change